### ISSUE TITLE
Internal Error while attempting to send a mail

### ISSUE THREAD

**Issue by severin-lemaignan** [mailpile/Mailpile#1088]
I got an white "Internal error" screen after pressing 'Send'.

Happen with a fresh Mailpile install (Oct 31, de50344610bb4), after setting up a single sending route via GMail (the UI says the credential are fine).

In the CLI, I simply get:
```
http://192.168.0.13:33411 "GET /message/draft/=D/?to=severin.lemaignan%40epfl.ch&cc=&bcc=&subject=Toto+est+en+vacances&body=Toto%0D%0A%0D%0A%0D%0A--+%0D%0ASent+using+Mailpile%2C+Free+Software+from+www.mailpile.is&attach-pgp-pubkey=yes&send=send&from=S%C3%A9verin+Lemaignan+%3Cseverin%40guakamole.org%3E&mid=D&encryption=none HTTP/1.1" 500 14
```

**Comment by BjarniRunar** [comment]
Closing this because there is not enough detail here to reproduce (or to verify whether it has been fixed). Thanks for filing though, perhaps give the Beta III a try later this summer?

**Comment by JocelynDelalande** [comment]
@severin-lemaignan should have been fixed by #1083, could you test please ?

**Comment by severin-lemaignan** [comment]
I'll try to test at some point, I'll reopen it if necessary.
